config/jobs/kubernetes/kops/kubernetes-presubmits.yaml

```diff
@@ -28,12 +28,12 @@ presubmits:
         - "--repo=k8s.io/release"
         - "--service-account=/etc/service-account/service-account.json"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
-        - "--timeout=75"
+        - "--timeout=200"
         - --scenario=kubernetes_e2e
         - --
         - --aws
         - --aws-cluster-domain=test-cncf-aws.k8s.io
-        - --build=bazel
+        - --build=release
```

Comment:
please do not cut a full release build in presubmit testing (!)
these are _wildly_ expensive

Reply:
further, we use bazel in presubmit for caching for a reason.